### DISCUSSION TITLE
How do I blacklist servers?

### DISCUSSION THREAD

**Discussion by ghost** [Vedza/NitroSniperGo#298] [Q&A] [answered]
Hi again, just wanted to ask how do I blacklist servers? ("blacklist_servers": []) <-- Does there need to be "" and , before putting another id? thanks

**Comment by Vedza** [comment, marked as answer]
Hi, 
Yes you need to put `"` and `,`
`"blacklist_servers": ["id1","id2","id3"]`

**Reply by ghost** [reply]
thanks
